source/query-with-natural-language/prompt-natural-language-query.txt

```diff
@@ -9,6 +9,99 @@ Prompt your First Natural Language Query
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 2
+   :depth: 1
 
-Prompt your First Natural Language Query
\ No newline at end of file
+You can use MongoDB Compass to generate filter documents using the
+natural language querying feature. This feature automatically generates 
+filter documents based on prompts you provide to the MongoDB AI engine. 
+Natural language querying serves as a helpful starting point and assists
+you in writing MongoDB queries.
+
+About this Task
+---------------
+
+.. include:: /includes/fact-natural-language-query.rst
+
+The MongoDB AI engine can return both filter documents and 
+aggregations. If your prompt results in an aggregation, a popup
+appears directing you to the :guilabel:`Aggregations` tab.
+
+This page uses the :ref:`sample_mflix.movies <sample-mflix>` 
+collection from the Atlas sample dataset.
+
+Before you Begin
+----------------
+
+You must :ref:`enable the natural language querying feature<enable-natural-language-querying>`.
+
+Steps
+-----
+
+.. procedure::
+   :style: connected
+
+   .. step:: Select the :guilabel:`Documents` tab
+
+   .. step:: Click the :guilabel:`Ask AI` button
+
+      The :guilabel:`Natural Language Query Bar` appears.
+
+   .. step:: Type a natural language prompt for your collection
+
+      Enter a prompt in the :guilabel:`Natural Language Query Bar`.
+      For example: ``Which movies were released in 2000?``.
+
+   .. step:: Press enter or click :guilabel:`Generate`
+
+      A filter document populates in the :guilabel:`Filter` bar.
+
+   .. step:: Carefully examine the filter document
+
+      Before running the query, make sure to thoroughly 
+      review the syntax.
+
+   .. step:: Click the :guilabel:`Find` button to execute the query
+
+      The results populate in the documents view.
```

Comment:
(suggestion -- step titles) 
 
Maybe something like:
3. Run the query
   a. Carefully examine the filter document. 
   b. Click Find to run your query. 
   The results populate in the documents view.

Reply:
Updated.